process/smartContract/process.go

```diff
@@ -621,6 +665,24 @@ func (sc *scProcessor) resolveFailedTransaction(
 	return process.ErrFailedTransaction
 }
 
+func (sc *scProcessor) computeBuiltInFuncGasUsed(
+	tx data.TransactionHandler,
+	gasProvided uint64,
+	gasRemaining uint64,
+) (uint64, error) {
+	_, txTypeOnDst := sc.txTypeHandler.ComputeTransactionType(tx)
+	if txTypeOnDst != process.SCInvoking {
+		return safeSubUint64(gasProvided, gasRemaining)
+	}
+
+	builtInFuncGasUsed := uint64(0)
```

Comment:
no need for this set. You can put directly := on L680

Reply:
done